packages/api/src/controllers/stream.ts

```diff
@@ -892,6 +892,26 @@ app.put(
     userId: user.id,
   });
 
+  if (req.body.active === false && stream.record === true) {
```

Comment:
WDYT of having a separate helper that emits all these stream active events? `emitStreamActiveEvents`?